Ghidra/Processors/AARCH64/data/languages/AARCH64-golang.cspec

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
there is a mixture of "-" and "_" as the separator in the filenames.  For the AARCH stuff, most of the other other files are using "_", so please rename this file to AARCH64_golang.cspec